src/System.IdentityModel.Tokens.Jwt/ClaimTypeAccessor.tt

```diff
@@ -0,0 +1,196 @@
+﻿<#@ template debug="false" hostspecific="false" language="C#" #>
```

Comment:
A couple of general concerns:
1. some of the categories seems to be AAD specific, IdentityModel has always tried to be neutral?
2. can users set their own categories?
3. there is no general spec for the category of claims, how can one say claim 'a' is in a category in general?

Reply:
1. Do we want to move this into SAL?
2. The categories are privacy categories (well defined. could almost be an enum). For 3) will follow-up offline